functions-methods.js

```diff
@@ -33,4 +63,17 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity (emailadres) {
+    if (emailadres.includes("@")) {
+        if (((emailadres.lastIndexOf(".")) === emailadres.length - 1) || (emailadres.includes(","))){
+            return false
+        } return true
```

Comment:
Ik raad je hier aan de condities die je controleert eerst in een variabel op te slaan, bijvoorbeeld zo:

`const containsAt = emailadres.includes("a");
const containsComma = emailadres.includes(",");
const containsNoDot = emailaddres.lastIndexOf('.') !== emailaddress.length - 1;

return !!(containsAt && !containsComma && containsNoDot);
`

Een if statement geeft in principe alleen een true of false waarde en met de laatste return kunnen we deze de dubbele uitroepteken, wat gelijk staat aan een if statement! ^^

De functie kan zelfs nog korter, maar het is nu goed om even de variabelen uit te schrijven :)